proto/cosmos/kv/kv.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+package cosmos.kv;
```

Comment:
Not sure this needs a new package to itself, but I guess it's okay...

Reply:
i can change, let me know what you prefer